xtp/include/votca/xtp/pmlocalization.h

```diff
@@ -32,18 +32,48 @@ class PMLocalization {
   PMLocalization(Logger &log, const tools::Property &options) : log_(log) {
     nrOfIterations_ = options.get(".max_iterations").as<Index>();
     convergence_limit_ = options.get(".convergence_limit").as<double>();
+    method_ = options.get(".method").as<std::string>();
   };
   void computePML(Orbitals &orbitals);
+  void computePML_UT(Orbitals &orbitals);
+  void computePML_JS(Orbitals &orbitals);
 
  private:
   Logger &log_;
+
+  std::string method_;
+
+  // functions for unitary optimizer
+  double cost(const Eigen::MatrixXd &W,
+              const std::vector<Eigen::MatrixXd> &Sat_all, const Index nat);
+  std::pair<double, Eigen::MatrixXd> cost_derivative(
+      const Eigen::MatrixXd &W, const std::vector<Eigen::MatrixXd> &Sat_all,
+      const Index nat);
```

Comment:
I think you can mark these functions const by adding `const` before the semicolon